hs_core/hydroshare/hs_bagit.py

```diff
@@ -141,6 +143,7 @@ def create_bag_files(resource):
                 hs_url=current_site_url,
                 res_id=resource.short_id,
                 file_name=f.short_path)
+            res_uri = encode_resource_url(res_uri)
```

Comment:
The warnings in the logs for unencoded urls are from creating the resourcemap here and below.